frontend/src/EncounterList.tsx

```diff
@@ -1,160 +1,200 @@
     root: {
-        width: '100%',
+      width: "100%",
     },
     paper: {
-        width: '100%',
-        marginBottom: theme.spacing(2),
+      width: "100%",
+      marginBottom: theme.spacing(2),
     },
     table: {
-        minWidth: 750,
+      minWidth: 750,
     },
     visuallyHidden: {
-        border: 0,
-        clip: 'rect(0 0 0 0)',
-        height: 1,
-        margin: -1,
-        overflow: 'hidden',
-        padding: 0,
-        position: 'absolute',
-        top: 20,
-        width: 1,
+      border: 0,
+      clip: "rect(0 0 0 0)",
+      height: 1,
+      margin: -1,
+      overflow: "hidden",
+      padding: 0,
+      position: "absolute",
+      top: 20,
+      width: 1,
     },
-});
+  });
 
 interface EncounterRow {
-    encounterId: string;
-    patient: string;
-    advocate: string;
-    time: number;
-    encounter_state : string;
-    patient_connected : boolean;
-    advocate_connected : boolean;
+  encounterId: string;
+  patient: string;
+  advocate: string;
+  time: number;
+  encounter_state: string;
+  patient_connected: boolean;
+  advocate_connected: boolean;
 }
 
-interface EncounterListProps extends RouteComponentProps<{}>, WithStyles<typeof styles> {
-    orderBy: string;
-    order: string;
-    user: firebase.User | null;
-    refresh: boolean; // Used to force refresh from server.
-    onVisit: (encounterId: string) => void;
+interface EncounterListProps
+  extends RouteComponentProps<{}>,
+    WithStyles<typeof styles> {
+  orderBy: string;
+  order: string;
+  user: firebase.User | null;
+  refresh: boolean; // Used to force refresh from server.
+  onVisit: (encounterId: string) => void;
 }
 
 interface EncounterListState {
   encounters: Array<EncounterRow>;
 }
 
 interface ListEncounterEntry {
-    encounterId: string;
-    encounter: {
-        patient: string;
-        when: number;
-    }
+  encounterId: string;
+  encounter: {
+    patient: string;
+    when: number;
+  };
 }
 
-class EncounterListImpl extends React.Component<EncounterListProps, EncounterListState>  {
-    constructor(props: EncounterListProps) {
-        super(props);
-        console.log("EncounterListImpl");
-        this.state = {encounters: []}
-    }
-
-    private headers = [
-        { id: 'name', numeric: false, disablePadding: true, label: 'Patient Name' },
-        { id: 'advocates', numeric: false, disablePadding: false, label: 'Advocates' },
-        { id: 'time', numeric: false, disablePadding: false, label: 'Scheduled Time' },
-        { id: 'encounter_state', numeric: false, disablePadding: false, label: 'State' },
-        { id: 'patient_connected', numeric: false, disablePadding: false, label: 'Patient Connected' },
-        { id: 'advocate_connected', numeric: false, disablePadding: false, label: 'Advocate Connected' },
-        { id: 'go', numeric: false, disablePadding: false, label: 'Go' },
-      ];
+class EncounterListImpl extends React.Component<
+  EncounterListProps,
+  EncounterListState
+> {
```

Comment:
this belons on the previous line

Reply:
This is done by the automatic formatter.